src/workspace/views/EmbedCodeOverview.tsx

```diff
@@ -0,0 +1,312 @@
+import { IconName, type MenuItemInfo, MoreOptionsDropdown } from '@viaa/avo2-components';
+import { isEqual } from 'lodash-es';
+import React, { type FC, useCallback, useEffect, useState } from 'react';
+import { useQueryParams } from 'use-query-params';
+
+import FilterTable, {
+	type FilterableColumn,
+} from '../../admin/shared/components/FilterTable/FilterTable';
+import { FILTER_TABLE_QUERY_PARAM_CONFIG } from '../../admin/shared/components/FilterTable/FilterTable.const';
+import { DeleteObjectModal, type LoadingInfo } from '../../shared/components';
+import EmbedCodeFilterTableCell from '../../shared/components/EmbedCodeFilterTableCell/EmbedCodeFilterTableCell';
+import { CustomError } from '../../shared/helpers';
+import withUser, { type UserProps } from '../../shared/hocs/withUser';
+import { useDebounce } from '../../shared/hooks/useDebounce';
+import useTranslation from '../../shared/hooks/useTranslation';
+import { type EmbedCodeFilters, EmbedCodeService } from '../../shared/services/embed-code-service';
+import { ToastService } from '../../shared/services/toast-service';
+import {
+	EMBED_CODE_DEFAULTS,
+	type EmbedCode,
+	type EmbedCodeOverviewFilterState,
+	type EmbedCodeOverviewTableColumns,
+} from '../../shared/types/embed-code';
+import { TableColumnDataType } from '../../shared/types/table-column-data-type';
+import { ITEMS_PER_PAGE } from '../workspace.const';
+
+// Typings
+interface EmbedCodeOverviewProps {
+	numberOfItems: number;
+	onUpdate: () => void | Promise<void>;
+}
+
+enum EmbedCodeAction {
+	EDIT = 'EDIT',
+	COPY = 'COPY',
+	DUPLICATE = 'DUPLICATE',
```

Comment:
what's the difference between copy and duplicate?
maybe rename COPY => COPY_TO_CLIPBOARD
```suggestion
	COPY_TO_CLIPBOARD= 'COPY_TO_CLIPBOARD',
	DUPLICATE = 'DUPLICATE',
```